go/README.md

```diff
@@ -4,27 +4,70 @@ This builder runs the `go` tool (`go build`, `go test`, etc.)
 after placing source in `/workspace` into the `GOPATH` before
 running the tool.
 
-This functionality is not necessary if you're building using
+# Using `Golang` and [Go Modules](https://github.com/golang/go/wiki/Modules)
+
+This Builder (`gcr.io/cloud-builders/go`) is not necessary if you're building using
 [Go modules](https://github.com/golang/go/wiki/Modules), available
-in Go 1.11+, and you can **build with the standard
-[`golang`](https://hub.docker.com/_/golang) image on Dockerhub
-instead:**
+in Go 1.11+. You can **build** with the `golang` image (not a Builder) from [Dockerhub](hub.docker.com/library/golang) and Google's Container Registry [mirror](mirror.gcr.io/library/golang):
```

Comment:
"(not a Builder)" is a bit misleading; any image can be used as a "builder" -- the term is fairly ill-defined -- including the `golang` image.

Reply:
Fixed: https://github.com/GoogleCloudPlatform/cloud-builders/pull/517/commits/f7fbd8a23bdb6a3be7e5911df90506958a525852